Makefile

```diff
@@ -1,3 +1,4 @@
+include .env
```

Comment:
What's the addition/benefit of adding the .env file to `Makefile`?

Reply:
To give it access to variables located in the .env and avoid writing the username/email directly in the Makefile. This way, all configuration elements remain centralized in the .env file.
Following the Django documentation, for non-interactive createsuperuser process, you need to specify --username and --email argument (With the desired values) inside the docker-compose command